dkist/tests/test_benchmarks.py

```diff
@@ -0,0 +1,17 @@
+import gzip
+from pathlib import Path
+
+from dkist import load_dataset
+from dkist.data.test import rootdir
+
+
+def load_asdf_from_gzip(tmp_path_factory):
+    vispdir = tmp_path_factory.mktemp("data")
+    with gzip.open(Path(rootdir) / "large_visp.asdf.gz", mode="rb") as gfo:
+        with open(vispdir / "test_visp.asdf", mode="wb") as afo:
+            afo.write(gfo.read())
```

Comment:
presumeably we would want this bit to be in a fixture so it's not part of the code getting benchmarked?